docs/build/ai-assistant.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Using the AI Assistant
+sidebar_label: AI Assistant
+---
+
+The OpenFn AI Assistant provides a chat interface with an AI Model to help you
+build workflows.
+
+You can use it to draft, proofread and debug job code, diagnose errors, and
+understand the capabilities of the platform.
+
+![AI Assistant](/img/ai-assistant.png)
+
+:::caution Assistant Unavailable?
+
+The AI Assistant must be configured before it can be used. If it's not available on your deployment, contact your system administrator.
```

Comment:
I'm not at all sure about the language here (especially the second line). Can anyone give me a hand?

Reply:
expanded this language a bit, and linked to our pricing page